.github/workflows/bokeh-ci.yml

```diff
@@ -4,38 +4,46 @@ on: [pull_request]
 
 jobs:
   build:
-    runs-on: ${{ matrix.os }}
-    strategy:
-      matrix:
-        python_version: [3.7]
-        os: [windows-latest, macos-latest]
+    runs-on: ubuntu-latest
+    env:
+      CONDA_REQS: "conda=4.7.12 conda-build=3.18.10 conda-verify=3.4.2 ripgrep=0.10.0"
```

Comment:
btw. What's the purpose for `ripgrep` in the build? It fails to install on Windows (at least 32-bit) and isn't mentioned anywhere explicitly except for `meta.yaml`.

Reply:
supposedly an optional dep that makes some aspect of conda quite faster.